packages/core/content-manager/admin/src/preview/pages/Preview.tsx

```diff
@@ -193,7 +206,11 @@ const ProtectedPreviewPageImpl = () => {
       zIndex={2}
       background="neutral0"
     >
-      <Page.Protect permissions={permissions}>
+      <Page.Protect
+        permissions={permissions.filter((permission) =>
+          permission.action.includes('explorer.read')
+        )}
+      >
```

Comment:
I'm not able to edit the form and I think it's caused by this part. Aren't you filtering out all permissions except read?

Reply:
Are the fields disabled for you? they don't look disabled on your slack screenshot

Reply:
yeah because I adjusted this line to pass all the permissions 🙃 


https://github.com/user-attachments/assets/30c274de-18dc-4e2a-835a-3a91d9efe2c5

Reply:
My thinking behind this: Page.Protect is the thing that prevents the page from being rendered at all. I only want that when users don't have read permissions, hence the filter

Disabling the fields though is done in InputRenderer, which relies on the DocumentRBAC context just below this. I do want to give this one the update permissions data so it can know whether to disable the fields

Reply:
Thanks I understand it better now and I see what you mean, but I think Page.Protect forwards the permissions it received to its children. So DocumenRBAC is only going to end up with read permissions

Reply:
https://github.com/strapi/strapi/blob/743192159941b6524bfa234b13e9d58b3d111710/packages/core/admin/admin/src/components/PageHelpers.tsx#L237-L241

Reply:
ah you're right, I didn't realize it was using the render prop. will fix now